go/vt/vtgate/grpcvtgateservice/server.go

```diff
@@ -262,17 +286,22 @@ func (vtg *VTGate) StreamExecute(request *vtgatepb.StreamExecuteRequest, stream
 	if err != nil {
 		return vterrors.ToGRPC(err)
 	}
-	vtgErr := vtg.server.StreamExecute(ctx,
-		string(request.Query.Sql),
-		bv,
-		request.KeyspaceShard,
-		request.TabletType,
-		request.Options,
-		func(value *sqltypes.Result) error {
-			return stream.Send(&vtgatepb.StreamExecuteResponse{
-				Result: sqltypes.ResultToProto3(value),
-			})
+	// Handle backward compatibility.
```

Comment:
Not worth it. Also, it's backward compatibility code; We'll be deleting it soon.